account_financial_report_qweb/__manifest__.py

```diff
@@ -16,7 +16,8 @@
     "website": "https://odoo-community.org/",
     'depends': [
         'account',
-        'account_group',    # account-financial-tools
+        'account_group',                        # account-financial-tools
+        'account_invoice_supplier_ref_unique',  # account-invoicing
```

Comment:
You can't change dependencies in a stable version, and more adding this dependency that is not mandatory. You must do a bridge module if you want that data.